PhotoCollection/PhotoCollection/View Controllers/PhotosCollectionViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  PhotosCollectionViewController.swift
+//  PhotoCollection
+//
+//  Created by Nathanael Youngren on 1/16/19.
+//  Copyright © 2019 Nathanael Youngren. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
+
+class PhotosCollectionViewController: UICollectionViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.collectionView!.register(UICollectionViewCell.self, forCellWithReuseIdentifier: reuseIdentifier)
```

Comment:
If you are registering a cell in storyboard, you don't need this line. Otherwise you will have two cells per row and also sometimes it cause crashes
```suggestion

```